hail/src/main/scala/is/hail/expr/ir/TableIR.scala

```diff
@@ -332,14 +332,15 @@ object LoweredTableReader {
         }
       }
     } else if (pksorted) {
```

Comment:
I know you didn't change it in this PR, but I'm confused why `pksorted` requires the max pk of one partition to *equal* the min pk of the next. Shouldn't that be LTEQ? Similarly for `ksorted`.

Reply:
good catch. This actually explains something I saw in the tests but didn't dig into -- some VCFs that ostensibly are sorted were throwing errors about unsorted not being supported!

Reply:
I'll PR that separately.